package.json

```diff
@@ -10,7 +10,7 @@
     "test:coveralls": "cat coverage/lcov.info | node node_modules/.bin/coveralls",
     "build": "tsc",
     "prod": "yarn run build && webpack --config prod.webpack.config.js && pm2 start pm2-prod.config.js",
-    "dev": "webpack --progress --config dev.webpack.config.js; tsc; webpack --watch --progress --config dev.webpack.config.js | node dist/server.js",
+    "dev": "webpack --progress --config dev.webpack.config.js && tsc && webpack --watch --progress --config dev.webpack.config.js && node dist/server.js",
```

Comment:
this does not work in ubuntu for me :(. If we change the pipe to &&, the server would only run after webpack is complete, and because it is in watch mode it is never complete.

Does it break windows if we keep the pipe? I will boot up my windows machine and test.

Reply:
btw, I think there _must_ be a better way to do this, it seems very hacky right now. I will have a look on how create-react-app makes it work.

Reply:
I was working on a better solution that uses npm-run-all instead of pipes and &&, but I didn't get the chance to test.

Reply:
I tested in my windows machine "webpack --progress --config dev.webpack.config.js && tsc && webpack --watch --progress --config dev.webpack.config.js | node dist/server.js" (with the pipe) and it worked ! I will commit it as a stop-gap solution. Thanks @hoovercj for helping :)